storm-core/src/jvm/backtype/storm/windowing/TimeEvictionPolicy.java

```diff
@@ -21,21 +21,30 @@
  * Eviction policy that evicts events based on time duration.
  */
 public class TimeEvictionPolicy<T> implements EvictionPolicy<T> {
-    private final long duration;
+    private final int windowLength;
+    protected Long currentTs;
```

Comment:
Its the reference time for the window calculation, based on which events are expired from the window. Will rename to `referenceTime`.